components/apps.js

```diff
@@ -516,11 +530,12 @@ SteamUser.prototype._getLicenseInfo = async function() {
 		return;
 	}
 
-	let packageids = this.getOwnedPackages();
+	// Get all owned lisense id's
+	let packageids = this.licenses.map(license => license.package_id);
 	let result;
 
 	try {
-		result = await this.getProductInfo([], packageids, false, undefined, PICSRequestType.Licenses);
+		result = await this.getProductInfo([], packageids, true, undefined, PICSRequestType.Licenses);
```

Comment:
If we skip requesting tokens, we may end up not returning owned apps, which could have bad consequences. This was this PR's original bugfix.